packages/pluggableWidgets/background-image-native/e2e/specs/background-image.spec.ts

```diff
@@ -1,86 +1,83 @@
-import { expectToMatchImageSnapshot, Widget, NativeHomePage, Alert } from "../../../../../tests/e2e";
-import { expect, waitFor, device } from "detox";
+import { expectToMatchScreenshot, tapMenuItem } from "../../../../../detox/src/helpers";
+import { Alert } from "../../../../../detox/src/Alert";
+import { expect, element, by, waitFor } from "detox";
 
 describe("Background image", () => {
     beforeEach(async () => {
-        await NativeHomePage().goToWidgetsHomePage();
+        await tapMenuItem("Background image");
+    });
 
-        const backgroundImagePage = Widget("btnBackgroundImage").getElement();
-        await backgroundImagePage.tap();
+    afterEach(async () => {
```

Comment:
Basically, we want use afterEach after every test. Unless we do no (inter)actions. If nothing on the screen changes, then there is also no reason to reset it's state. Then it's fine to use afterAll. afterAll is faster since we don't need to restart.